pkg/snapshot/snapshotter/snapshotter.go

```diff
@@ -234,10 +246,12 @@ func (ssr *Snapshotter) takeDeltaSnapshot() error {
 	}
 	// compute hash
 	hash := sha256.New()
-	hash.Write(data)
+	if _, err := io.Copy(hash, bytes.NewReader(data)); err != nil {
```

Comment:
Does `io.Copy()` provide any significant advantage over `Write()` ?

Reply:
Hmm, right. It won't in this case. Since `hash.Write` is actually does in memory computation over byte array and there isn't network/disk IO involved.  I Will rollback but do add missing error handling.